indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -156,6 +157,13 @@
                                         {{- friendly_data if friendly_data else '-' -}}
                                     </td>
                                 {% endfor %}
+                                <td class="i-table">
+                                    {% if abstract.state.name == 'accepted' and abstract.contribution%}
```

Comment:
space before `%}`.

Also check if you need to add `subqueryload('contribution')` to the query options to avoid spamming one query per accepted abstract